02_deploying_to_kubernetes.md

```diff
@@ -269,7 +271,11 @@ kubectl explain deployment.spec.replicas
 
 **Now that you know how to look up the documentation of Kubernetes resources, let's turn back to the Deployment.**
 
-The first four lines define the type of resource (Deployment), the version of this resource type (`apps/v1`), and the name of this specific resource (`knote`):
+The first four lines define: 
+
+- the type of resource (Deployment)
+- the version of this resource type (`apps/v1`), and 
+- the name of this specific resource (`knote`)
```

Comment:
In general, for bullet points:

- The first letter is a capital letter.
- The bullet point ends with a dot.